conda/build.sh

```diff
@@ -0,0 +1,5 @@
+pip install -e .
+
```

Comment:
There's a step missing here to copy the idl files installed from the yum repo so they are packaged together. You can accomplish this by adding the following here:

```

cp -v /opt/lsst/ts_sal/idl/*idl `python -c 'from lsst.ts import idl; print(idl.get_idl_dir())'`

```

Reply:
Fixed